src/translations/pages/contact.ts

```diff
@@ -0,0 +1,23 @@
+import { translation as _ } from '@atb/translations/commons';
+
+export const Contact = {
+  title: _(
+    'Hva kan vi hjelpe deg med?',
+    'What can we help you with?',
+    'Kva kan vi hjelpe deg med?',
+  ),
+  ticketControl: _(
```

Comment:
Maybe we should structure this so that all translations related to `ticketControl` is within a `ticketControl` object? Something like:

```ts
...
ticketControl: {
 title: _('Billettkontroll og gebyr', 'Ticket control and fee', 'Billettkontroll og gebyr'),
},
travelGuarantee: {
 title: _('Reisegaranti', 'Travel guarantee', 'Reisegaranti'),
}
...

```